behat-tests/features/bootstrap/FeatureContext.php

```diff
@@ -400,12 +424,14 @@ public function assertLinkVisibleRegion($link, $region) {
   }
 
   /**
-   * @Then I should not see the link :link visible in the :region( region)
+   * @Then I should not visibly see the link :link in the :region( region)
```

Comment:
There is no Behat standard for defining these step language. However, we do have a similar step in our imported Context, MinkContext, ```Gherkin @Then I should not visibly see the link :link```, which we can also use in our feature files in the future.

After aligning the language, we now have:
```Gherkin
# from MinkContext.php
@Then I should not visibly see the link :link
# from FeatureContext.php
@Then I should not visibly see the link :link in the :region( region)
```